main.go

```diff
@@ -24,6 +25,7 @@ func run() error {
 	if err != nil {
 		return fmt.Errorf("new gitops config: %w", err)
 	}
+	stepconf.Print(cfg)
```

Comment:
Do we still need this?

Reply:
Yes, we always print the step config, in all of our steps

Reply:
Good to know, thanks